core/src/main/scala/org/http4s/HttpDate.scala

```diff
@@ -54,6 +55,11 @@ class HttpDate private (val epochSecond: Long) extends Renderable with Ordered[H
 object HttpDate {
   private val MinEpochSecond = -2208988800L
   private val MaxEpochSecond = 253402300799L
+  implicit val catsOrderForHttp4sHttpDate: Order[HttpDate] =
```

Comment:
@san-coding if you are going to add add an `Order` can you also add a `Hash` with it? Something like we do here https://github.com/http4s/http4s/blob/main/core/shared/src/main/scala/org/http4s/RequestPrelude.scala#L102

Sorry, this is extra confusing and weird. The reason we have to do this is that both [Order](https://github.com/http4s/http4s/blob/main/core/shared/src/main/scala/org/http4s/RequestPrelude.scala#L102) and [Hash](https://github.com/typelevel/cats/blob/e0784fc0d95b9c4eed0bb1ddce6c69e3fd9a4a4f/kernel/src/main/scala/cats/kernel/Hash.scala#L12) extend [Eq](https://github.com/typelevel/cats/blob/e0784fc0d95b9c4eed0bb1ddce6c69e3fd9a4a4f/kernel/src/main/scala/cats/kernel/Eq.scala#L18) and if they aren't both in the concrete instance we get into an ambiguous implicit context.

So, if we don't add a `Hash` instance at the same time we add an `Order` we'll have to do a deprecation in the future if we ever want to add a `Hash`.

I hope that makes sense....I'm sorry, it's very confusing.

Reply:
Ok I will add Hash, but can you tell me what I should put inside the instance definition 
```
 implicit val catsOrderForHttp4sHttpDate: Order[HttpDate] =
    new Order[HttpDate] {
    What should add here
    }
```